src/js/components/modals/UpdateConfigModalContents.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+
+import AdvancedConfig from '../AdvancedConfig';
+import CosmosErrorMessage from '../CosmosErrorMessage';
+import CosmosPackagesStore from '../../stores/CosmosPackagesStore';
+import Service from '../../structs/Service';
+import ServicePlan from '../../structs/ServicePlan';
+import UniversePackage from '../../structs/UniversePackage';
+
+const METHODS_TO_BIND = [
+  'getConfigSubmit',
+  'handleAdvancedFormChange',
+  'handleConfigSave'
+];
+
+class UpdateConfigModal extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      hasFormErrors: false
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  handleAdvancedFormChange(formData) {
+    this.setState({hasFormErrors: !formData.isValidated});
+  }
+
+  handleConfigSave() {
+    let serviceMetadata = this.props.service.getMetadata();
+    let {name, version} = serviceMetadata;
```

Comment:
You're assigning a value to `serviceMetadata` just to assign `serviceMetadata` to other variables.
Do we need `serviceMetadata`?